### PR TITLE
MLFlow Ubuntu 22.04 & Python 3.12 environment

### PR DESCRIPTION
**Pull request by Tarun-Chevula**
Updated Ubuntu 22.04 & Python 3.12

### CODE REVIEW

File: assets/inference/environments/mlflow-ubuntu20.04-py39-cpu-inference/context/conda_dependencies.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- python=3.9.13
+- python=3.12
```
Comment (vizhur) [inline assets/inference/environments/mlflow-ubuntu20.04-py39-cpu-inference/context/conda_dependencies.yaml:6]: python version of environment should match the name.

**Review by vizhur — CHANGES_REQUESTED** [review]
you might want to create a new environment for py312

File: assets/inference/environments/mlflow-ubuntu20.04-py39-cpu-inference/context/conda_dependencies.yaml (hunk 1 above)
Comment (Tarun-Chevula) [reply assets/inference/environments/mlflow-ubuntu20.04-py39-cpu-inference/context/conda_dependencies.yaml]: Thanks for the catch. This ia a new environment.

**Review by vizhur — APPROVED** [review]

**Comment by vizhur** [comment]
This PR contains a brand new asset and requires the 'safe to publish' label to run this check. Assets must be properly evaluated before being added Github and the public registry. Please reach out to System Registry Content team to review this PR.

This error means that you ask system registry content team to review the PR and set the label if needed. Any PR merged with self-set label will be reverted with no warning.

**Pull request merged**